src/routes/auth.js

```
@@ -32,4 +36,29 @@
+  try {
+    console.log("JWT_CONFIRM_SECRET:", jwt.verify(token, process.env.JWT_CONFIRM_SECRET));
+    const decoded = jwt.verify(token, process.env.JWT_CONFIRM_SECRET);
+    
+
+    const findUser = await user.findById(decoded.id);
+    if (!findUser) return res.status(404).json({ message: 'User not found' });
+
+    findUser.isEmailConfirmed = true;
+    await findUser.save();
+
+    res.redirect('http://localhost:3000');
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)